includes/class-evf-form-task.php

```diff
@@ -103,8 +103,8 @@ public function do_task( $entry ) {
 			}
 
 			// reCAPTCHA check.
-			$site_key   = get_option( 'evf_recaptcha_site_key', '' );
-			$secret_key = get_option( 'evf_recaptcha_site_secret', '' );
+			$site_key   = get_option( 'everest_forms_recaptcha_site_key', '' );
+			$secret_key = get_option( 'everest_forms_recaptcha_site_secret', '' );
 			if (
 				! empty( $site_key ) &&
 				! empty( $secret_key ) &&
```

Comment:
`'1' == $form_data['settings']['recaptcha_support']` check looks redundant. `isset()` only should've done.

Reply:
@sanzeeb3 This is needed because that `1` is received as string not boolean value :)